### ISSUE TITLE
Align entries_data rows to SQLite page size

### ISSUE THREAD

**Issue by mtlynch** [mtlynch/picoshare#223]
SQLite's default page size is [4096 bytes](https://www.sqlite.org/pgszchng2016.html).

There's a perf bottleneck in `entries_data` because PicoShare has to read/write many rows per file, especially if the file is large. We could optimize this by tuning the [chunk size](https://github.com/mtlynch/picoshare/blob/50e88c84b8dd82c697c651d50319966fb2e6c00b/store/sqlite/sqlite.go#L22) so that the total size of each row is a multiple of 4096. That would let SQLite read exactly the data it needs from disk without having to read an extra page.

Credit: [Nicolas Grilly](https://twitter.com/ngrilly/status/1520428549815119873)

**Comment by marshalleq** [comment]
Yeah this kind of optimisation is extremely important, but most people don't understand its value.  Thanks for raising it.
